src/devhostagent/PortForward/ServicePortForwardConnector.cs

```diff
@@ -196,25 +197,35 @@ private async Task HookupStreamData(StreamHookData s)
         /// </remarks>
         private byte[] GetModifiedBytesForManagedIdentity(byte[] data)
         {
-            const string headerToInsert = "Metadata: true\n";
-            var stringContent = Encoding.Default.GetString(data.ToList().ToArray());
+            const string headerToInsert = "Metadata: true";
+            var stringContent = Encoding.Default.GetString(data);

```

Comment:
💡 Please note: I am not entirely sure of this change, but adding some thoughts: `GetString()` take byte array we are changing the data contract from what it used to be to `data` so is this contract changed recently?

Reply:
this is my attempt at simplifying this code. I have tested with both ways and there is no regression. data is a byte array. I do not see the need to go from bytes array to list and then to array again if the GetString can work with data directly.